### ISSUE TITLE
FTP question

### ISSUE THREAD

**Issue by klundry** [mwoz123/send2ebook#1]
Can this be used with SFTP (over ssh)? So, can I just edit the config file and use a server that I already have ssh access to without setting up an ftp server?

**Comment by mwoz123** [comment]
Just checked in docs from apache ftp and it requires different class - http://commons.apache.org/proper/commons-net/apidocs/org/apache/commons/net/ftp/FTPSClient.html

I should be able to change this quickly in notepad but currently don't have set up SDK for testing/building package. Would you be able to build it by your own?

**Comment by mwoz123** [comment]
Change in new branch: https://github.com/mwoz123/send2ebook/tree/sftp
Warn as above - it's untested.

**Comment by klundry** [comment]
I actually don't know how to build it. Do I need a java sdk?

**Comment by mwoz123** [comment]
Yes jdk is required. Hopefully I'll find time this week to build it..
